content/docs/tasks/traffic-management/custom-gateway/index.md

```diff
@@ -0,0 +1,359 @@
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+      volumes:
+      - name: istio-certs
+        secret:
+          defaultMode: 420
+          optional: true
+          secretName: istio.istio-ingressgateway-service-account
+      - name: ingressgateway-custom-certs
+        secret:
+          defaultMode: 420
+          optional: true
+          secretName: istio-customingressgateway-certs
+      - name: ingressgateway-ca-certs
+        secret:
+          defaultMode: 420
+          optional: true
+          secretName: istio-ingressgateway-ca-certs
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: beta.kubernetes.io/arch
+                operator: In
+                values:
+                - amd64
+                - ppc64le
+                - s390x
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 2
+            preference:
+              matchExpressions:
+              - key: beta.kubernetes.io/arch
+                operator: In
+                values:
+                - amd64
+          - weight: 2
+            preference:
+              matchExpressions:
+              - key: beta.kubernetes.io/arch
+                operator: In
+                values:
+                - ppc64le
+          - weight: 2
+            preference:
+              matchExpressions:
+              - key: beta.kubernetes.io/arch
+                operator: In
+                values:
+                - s390x
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - istio-internal-custom
+            topologyKey: kubernetes.io/hostname
+{{< /text >}}
+
+Please Note that you have to force a NodePort to an available Port.
+
+You also have to declare your `ingressgateway-custom-certs` with secret name generated before (for example `secretName: istio-customingressgateway-certs`)
+
+## Create Istio gateway
+
+You can now create an Istio custom gateway:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  annotations:
+  name: istio-custom-gateway
+  namespace: default
+spec:
+  selector:
+    istio: istio-internal-custom
+  servers:
+  - hosts:
+    - '*.mydemo.com'
+    port:
+      name: http
+      number: 80
+      protocol: HTTP
+    tls:
+      httpsRedirect: true
+  - hosts:
+    - '*.mydemo.com'
+    port:
+      name: https
+      number: 443
+      protocol: HTTPS
+    tls:
+      mode: SIMPLE
+      privateKey: /etc/istio/ingressgateway-certs/tls.key
+      serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
+{{< /text >}}
+
+You can now use your custom gateway `istio-custom-gateway`
```

Comment:
Add a blank line at the end of the file. It helps avoid issues with some Markdown parsers. I suggest adding a congratulatory comment to encourage readers further:
```suggestion
**Congratulations!** You can now use your custom `istio-custom-gateway` gateway configuration object. 
```